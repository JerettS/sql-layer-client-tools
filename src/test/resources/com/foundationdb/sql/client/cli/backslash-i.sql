SELECT 'first';
\i select.inc
SELECT 'second';
\i recursive.inc
SELECT 'third';
\i quit.inc
SELECT 'fourth';
