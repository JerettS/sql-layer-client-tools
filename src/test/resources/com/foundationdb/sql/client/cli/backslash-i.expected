_SQL_COL_1 
------------
 first      
(1 row)

SELECT 2;
 _SQL_COL_1 
------------
          2 
(1 row)

SELECT 4;
 _SQL_COL_1 
------------
          4 
(1 row)

 _SQL_COL_1 
------------
 second     
(1 row)

SELECT 'rec_pre';
 _SQL_COL_1 
------------
 rec_pre    
(1 row)

\i select.inc
SELECT 2;
 _SQL_COL_1 
------------
          2 
(1 row)

SELECT 4;
 _SQL_COL_1 
------------
          4 
(1 row)

SELECT 'rec_post';
 _SQL_COL_1 
------------
 rec_post   
(1 row)

 _SQL_COL_1 
------------
 third      
(1 row)

SELECT 'quit_pre';
 _SQL_COL_1 
------------
 quit_pre   
(1 row)

\q

